doc/jetty-config.md

```diff
@@ -49,6 +49,13 @@ number of seconds, see http://man7.org/linux/man-pages/man7/socket.7.html.  Note
 that the effect of setting this option may vary depending upon the operating
 system's underlying implementation.
 
+### `idle-timeout-milliseconds`
+
+This optional setting can be used to control how long Jetty will allow a
+connection to be held open by a client without any activity on the socket.  If
+a connection is idle for longer than this value, Jetty will forcefully close
+it from the server side.  Jetty's default value for this setting is 30 seconds.
```

Comment:
I might include something about the following paragraph in the description, but I'll defer to your judgement...

From what I've seen, Jetty will only forcefully close the socket per the idle timeout if a client request is not being actively handled by the server.  For example, if the client had issued a full HTTP request, the server is still in the process of preparing a response, and no subsequent activity had taken place on the socket up to the configured idle timeout, Jetty will not forcefully close the connection from the server side.  Instead, Jetty will keep the socket alive from its side through the delivery of the response.  Once the response has been delivered, if no subsequent activity on the socket occurs up through the configured idle timeout again, Jetty would close the socket.

Jetty also uses the idle timeout as part of its internal algorithm for scaling up / down worker threads.  If one of its worker threads has been idle for a period up to the idle timeout, Jetty will let that thread die - so long as Jetty's worker thread pool has at least the configured minimum number of threads still available to it.  I'm thinking that the threading behavior may be more of an internal detail that's not worth trying to document here.